db_dtypes/__init__.py

```diff
@@ -341,6 +348,8 @@ def __sub__(self, other):
     "__version__",
     "DateArray",
     "DateDtype",
+    "JSONDtype",
```

Comment:
Could we omit these from `__all__` if they are `None`, please?

I'd like `hasattr(db_dtypes, "JSONDtype")` to correctly be `False` if `JSONDtype` is not supported. In general, this "duck typing" is preferred in Python over explicit version number checks.

Reply:
Good to known. Done.